metadetect/fitting.py

```diff
@@ -85,7 +84,7 @@ def go(self, mbobs_list):
         if len(datalist) == 0:
             return None
         else:
-            return eu.numpy_util.combine_arrlist(datalist)
+            return np.hstack(datalist)
```

Comment:
This change is fine, but I think combine_arrlist is more memory efficient than hstack. Not sure if that matters for us here though.

Reply:
I agree, but I did this so the code could be used without esutil, which is not a dep for the stack

Reply:
Of course this also depends on ngmix so we've already started down the rabbit hole

Reply:
Yeah. They are using conda now with the rubin-env metapackage: https://github.com/conda-forge/rubinenv-feedstock

So anything in conda-forge is easy for them to use now. I would not worry about this in the future. Happy to keep this change though.